src/js/frontend/README.md

```diff
@@ -15,7 +15,7 @@ Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 The page will reload if you make edits.\
 You will also see any lint errors in the console.
 
-### `yarn test`
+### `yarn integrationTest` or `yarn unitTest`
 
 Launches the test runner in the interactive watch mode.\
```

Comment:
Seems like this documentation paragraph is now out of date and needs to be updated.